README.md

```diff
@@ -748,6 +752,54 @@ The documentation on using Closure Stylesheets can be found
   `bazel run @io_bazel_rules_closure//closure/stylesheets -- --help`
 
 
+## closure\_proto\_js\_library
+
+```python
+load("@io_bazel_rules_closure//closure:defs.bzl", "closure_proto_js_library")
+closure_proto_js_library(name, srcs, add_require_for_enums, namespace_prefix,
+                         error_on_name_conflict, binary, import_style)
+```
+
+Defines a set of Protocol Buffer files.
+
+Documentation: [Protocol Buffers][protobuf] [JS][protobuf-js]
+[Generator Options](https://github.com/google/protobuf/blob/master/src/google/protobuf/compiler/js/js_generator.h).
+
+#### Implicit Output Targets
+
+- *name*.js: A generated protocol buffer JavaScript library.
+
+### Arguments
+
+- **name:** ([Name][name]; required) A unique name for this rule. Convention
+  states that such rules be named `foo_proto`.
+
+- **srcs:** (List of [labels][labels]; required) A list of `.proto` source
+  files that represent this library.
+
+- **add_require_for_enums:** (Boolean; optional; default is `False`) Add a
+  `goog.requires()` call for each enum type used. If false, a forward
+  declaration with `goog.forwardDeclare` is produced instead.
+
+- **namespace_prefix:** (String; optional; default is `None`) Specify this
+  to prefix the generated namespace.
+
+- **error_on_name_conflict:** (Boolean; optional; default is `True`) Errors
+  if there are two types that would generate the same output file.
+
+- **binary:** (Boolean; optional; default is `True`) Enable binary-format support.
+
+- **import_style:** (String; optional; default is `None`) Specifies the type
```

Comment:
Shouldn't default be `"IMPORT_CLOSURE"`?

Also side note: I'm torn about whether or not it would be worth it to automatically add the `IMPORT_` suffix for the user. I'll support you if you want to do that. But either way is fine. Might be better to follow same convention as the tool itself.